org.eclipse.winery.repository/src/main/java/org/eclipse/winery/repository/backend/filebased/AbstractFileBasedRepository.java

```diff
@@ -98,7 +99,15 @@ public AbstractFileBasedRepository(Path repositoryRoot) {
 
     public void forceDelete(RepositoryFileReference ref) throws IOException {
         Path relativePath = this.fileSystem.getPath(BackendUtils.getPathInsideRepo(ref));
-        Path fileToDelete = this.makeAbsolute(relativePath);
+        Path pathToDelete = this.makeAbsolute(relativePath);
+        if (Files.isDirectory(pathToDelete)) {
+            FileUtils.forceDelete(pathToDelete);
+        } else {
+            forceDeleteFile(pathToDelete, ref);
+        }
+    }
+
+    public void forceDeleteFile(Path fileToDelete, RepositoryFileReference ref) throws IOException {
```

Comment:
Do you need this distinction of the methods?

Reply:
Yes, the existing method was designed to only delete files as well as their mime type files. In case of a directory, the method provided by FileUtils is required.